.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,164 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
This file should not be commited. Only add those files to the commit in which you actively made changes that are part of the implementation of this feature, i.e. MediaPlayerService.java, settings.xml and strings.xml.

Reply:
I would suggest adding .idea to .gitignore. This keeps happening.

Reply:
I would suggest adding .idea to .gitignore? This seems to keep happning.